README.md

```diff
@@ -74,12 +79,20 @@ jobs:
       contents: read
 ```
 
-to ensure the changelog has been updated for any PR to `develop` or `main`. 
+to ensure the changelog has been updated for any PR to `develop` or `main`. For PRs that **do not** contain any changes
+that would require a version bump, and therefore **do not** need to be documented in the changelog, you can label the PR
+`bumpless` to skip this check.
 
 ### [`reusable-create-jira-issue.yml`](./.github/workflows/reusable-create-jira-issue.yml)
 
-Creates a Jira issue that corresponds to the labeled GitHub issue. Use like:
+When an issue is labeled with a `Jira ` prefixed label, a Jira work item will be created of the corresponding type. The
```

Comment:
```suggestion
When an issue is labeled with a `Jira ` prefixed label, a Jira issue will be created of the corresponding type. The
```

Reply:
@jtherrmann backlog is fine, but JIRA does refer to this as work items / work types:
https://support.atlassian.com/jira-cloud-administration/docs/what-are-issue-types/

so I used "work" to be in line with JIRA docs and searching for the available/default ones easier.

Reply:
Sounds good!